### PR TITLE
#165305258 Integrate Coveralls 

### PR DESCRIPTION
**Pull request by ElishaMisoi**
### What does this PR do?
-Integrates Coveralls code coverage service

### Description of Task(s) to be completed?

-Integrate Coveralls code coverage service

### How should this be manually tested?

Clone this repository on local machine and cd into project folder

`$ git clone git@github.com:andela/ah-the-jedi-backend.git`
`$ cd ah-the-jedi-backend`

Check out to the PR's branch

`$ git checkout ch-integrate-coveralls-165305258`

Create and activate virtual environment

`$ python3 -m venv venv`
`$ source venv/bin/activate`

Install app dependencies from requirements.txt

`$ pip install -r api/requirements.txt`

Check the versions of the installed packages

`$ pip freeze`

### What are the relevant Pivotal Tracker stories?
[#165305258](https://www.pivotaltracker.com/story/show/165305258)

### Screenshots

Screenshot 2019-04-23 at 16 19 32

### CODE REVIEW

**Comment by jkamz** [comment]
Hello. Nice work so far. Please include coveragerc file so that you can omit some files and folders which should not be included in the coverage. For instance migration files should not be included in the coverage.

**Comment by ElishaMisoi** [comment]
> Hello. Nice work so far. Please include coveragerc file so that you can omit some files and folders which should not be included in the coverage. For instance migration files should not be included in the coverage.

Done. Thanks for the review.

**Review by jkamz — APPROVED** [review]

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[report]
+omit = 
+    *migrations*
+    *tests*
+    config/*
+    test/*
+    manage.py
```
Comment (katherine95) [inline .coveragerc]: Add newline at the end of file.

**Review by katherine95 — APPROVED** [review]

**Review by AlvinMugambi — APPROVED** [review]
Nice work adding the .coveragerc

**Review by Benkimeric — COMMENTED** [review]
Nice implementation, however just a minor issue in your description on how to manually test -->
```
$ python3 -m venv venv
$ source .venv/bin/activate
```
you have created a virtual environment named `venv` but activating a `.venv` please change it to match

File: .coveragerc (hunk 1 above)
Comment (ElishaMisoi) [reply .coveragerc]: Fixed

**Comment by MisoiElisha** [comment]
> Nice implementation, however just a minor issue in your description on how to manually test -->
> 
> ```
> $ python3 -m venv venv
> $ source .venv/bin/activate
> ```
> 
> you have created a virtual environment named `venv` but activating a `.venv` please change it to match

Thanks for the note. I have made the change.

**Comment by ElishaMisoi** [comment]
> Nice implementation, however just a minor issue in your description on how to manually test -->
> 
> ```
> $ python3 -m venv venv
> $ source .venv/bin/activate
> ```
> 
> you have created a virtual environment named `venv` but activating a `.venv` please change it to match

Thanks for the note. I have made the change.

**Review by rnjane — APPROVED** [review]

**Pull request merged**